services/fill.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Start the fill station binary
+After=network.target
+
+[Service]
+Restart=always
+RestartSec=1
+User=root
+WorkingDirectory=/home/crt
+ExecStart=/home/crt/fill_station
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Can we make this consistent between the two? Either both default or both multi-user?

Reply:
Default should too. Sounds like default can have since issues though, so we should update both to be multi-user i think

Reply:
Sounds good, I'll make that change on ground.service then